scripts/fix-key-and-update.sh

```diff
@@ -19,7 +19,7 @@ curl https://raw.githubusercontent.com/ublue-os/main/main/cosign.pub > /etc/pki/
 
 # Ensure the path to the public key matches the local copy location.
 echo "Updating the path to the public key in the container policy..."
-sed -i.bak "s#/usr/etc/pki/containers/ublue-os.pub#/etc/pki/containers/ublue-os.pub#" /etc/containers/policy.json
+sed -i.bak "s#/etc/pki/containers/ublue-os.pub#/etc/pki/containers/ublue-os.pub#" /etc/containers/policy.json
```

Comment:
I think this is still needed.  It's pattern matching on the old string (containing /usr)

Reply:
Fixed, thanks